src/markdown-pages/docs/writing-own-tests/running-test-modes.md

```diff
@@ -0,0 +1,92 @@
+
+
+## Running a test locally
+
+
+When you run a test locally, without using shared configuration, everything runs on your local machine. The Galasa bootstrap file is blank and makes no reference to an ecosystem. The Galasa framework is launched within the JVM on the local machine and the local file system holds all the configuration that is used by the test. The test runs in the local JVM and all test results and artifacts are stored on the local disk. 
+
+![running in local mode:](running-local.svg)
+
+You can run a test in this mode by using the following command:
+
+```
+galasactl runs submit --local … <more options are needed>
+```
+
+or by using the Eclipse plugin, in which case the test is hosted within the Eclipse JVM.
+
+
+## Running a test locally but using shared configuration
+
+When you run a test locally, but using shared configuration, the Galasa bootstrap file refers to the Galasa Ecosystem that you want to use. The Galasa framework is launched within the JVM on the local machine, but the framework consults the remote ecosystem to read configuration data. This is the key difference between running a test locally without using shared configuration. The test still runs in the local JVM and all test results and artifacts are stored on the local disk. 
+
+![running in local mode with shared configuration:](running-hybrid.svg)
+
+You can run a test in this mode by setting up your bootstrap to refer to the ecosystem that you want to use and running the following command:
+
+```
+galasactl runs submit --local 
+```
+
+or by using the Eclipse plugin, in which case the test is hosted within the Eclipse JVM.
+
+
+### When to run a test locally
+
+Running a test locally is useful when you are doing the following types of task:
+
+- Developing tests, and running those tests against a non-production environment.
+
+- Running ad-hoc tests against a development or non-production target system.
+
+- Running many tests in series against a target system which can withstand the load that is placed on it.
+
+- Running tests against a target system which is dynamically provisioned, used exclusively by the tests, and de-provisioned afterwards.
+
+- Running tests within a JVM that will not be interrupted, or halted mid-flow, causing a build-up of allocated resources in the system under test.
+
+- When there is time to find and share test run logs and artifacts with bug investigators, whenever a test fails and finds a bug.
+
+
+## Running a test remotely in a Galasa Ecosystem
+
+When you run a test remotely, the Galasa bootstrap file refers to the Galasa Ecosystem that you want to use, and the test is run on the specified remote ecosystem.  The configuration for the test is also held within that ecosystem, and Galasa starts up on the container in which the test code is running. The test results and artifacts are stored in a database within the ecosystem and authorised users on client machines are able to view the test results. 
```

Comment:
| Galasa starts up on the container
should be 
'Galasa starts up in a container' ?
There are generally more than one containers which are running galasa tests in parallel.

Reply:
yes, reworded thank you